### ISSUE TITLE
[Accessibility]A11y_Java Tooling for IntelliJEclipse_Customize New Project_Keyboard: Tooltips for icons are visible only on hover over but not available with keyboard as focus is not going to the icons 

### ISSUE THREAD

**Issue by soumyatippapuram** [microsoft/azure-tools-for-java#3342]
[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier!

GitHubTags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_Java Tooling for IntelliJ/Eclipse-Stay Healthy; #MAS2.1.1; #Keyboard; #C+E-H; #Java tooling for IntelliJ/Eclipse;

**Environment Details:**
Download and Install IntelliJ IDEA from https://docs.microsoft.com/en-us/java/azure/intellij/azure-toolkit-for-intellij-create-hello-world-web-app?view=azure-java-stable
IntelliJ IDEA 2019.1.3
Build #IU-191.7479.19

**OS Details: ​**
OS: Windows (Version 1903, Build: 18362.175)
MAS Violated: MAS 2.1.1

**Note: Same issue is observed in all the screen that contain these icons**

**Repro Steps: ​**
1) Open IntelliJ IDEA application
2) Press 'Ctrl+Alt+S' to open application settings/preferences popup
3) Tab till 'Plugins' tab and press enter
4) Search for 'Azure Toolkit for IntelliJ' in the Marketplace and press enter.
5) Tab till 'Install' button and press enter
6) Restart IntelliJ IDEA application, and tab till Tools-->Azure-->Azure Sign in
7) Sign in with any Azure subscription
8) Tab till File menu and click new-->Project
9) In the New Project dialog box, select Maven and press enter
10) Check "Create from archetype" check box and select "maven-archetype-webapp" and tab till Next and press enter
11) Specify the GroupId and ArtifactId and press Next button
12) Check if the icon buttons (browse, file, Add, Remove, Edit) are accessible through keyboard

**Actual Result:**
Tooltips for icons are visible only on hover over but not available with keyboard as focus is not going to the icons

**Expected Result:**
Keyboard focus should go to the icons and tooltips should be visible 

**User Impact:**
Keyboard only users and screen reader users will not be aware of the shortcuts (as mentioned in the tooltips) if the focus doesn't go to those icons and tooltips are visible only on mouse over

**MAS Reference:**
[MAS 2.1.1](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2})

[Bug_7_IntelliJ_Customize_Icons_Keyboard.pptx](https://github.com/microsoft/azure-tools-for-java/files/3451295/Bug_7_IntelliJ_Customize_Icons_Keyboard.pptx)
![Bug_7_IntelliJ_Edit Configuration_Icons_Keyboard](https://user-images.githubusercontent.com/41108985/62205862-09b44000-b3ae-11e9-9b2d-a48c5d090453.png)
![Bug_7_IntelliJ_Finish_Icons_Keyboard](https://user-images.githubusercontent.com/41108985/62205863-09b44000-b3ae-11e9-84c2-4f17520609b2.png)

**Comment by Flanker32** [comment]
@soumyatippapuram Both the before launch panel and the new project dialog is the ui of IntelliJ, which is out of the scope of out toolkit.

**Comment by Flanker32** [comment]
* **Importance of experience**: Little, it not a common behavior to hover on these icons to get tooltip and we hadn't received any customer feedback about this issue.
* **Quality of experience**: Little, the icon could explain the what will do without a tooltip.
* **Alternate solutions**: No alternate technologies or options can be leveraged.
* **Evidence**: All evidence is provided in this issue and external issue.

**Comment by Flanker32** [comment]
@soumyatippapuram This is an external issue of IntelliJ which has been tracked in AAT, please help us review it, thanks!

**Comment by soumyatippapuram** [comment]
Reported this issue to IntelliJ (https://youtrack.jetbrains.com/issue/IDEA-222356)

**Comment by Flanker32** [comment]
Discussed this bug with Ryan and Sarah and approved to close this issue as its external and there is non blocking scenario.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749760
